test/parallel/test-tls-hello-parser-failure.js

```diff
@@ -57,6 +57,6 @@ const server = tls.createServer(options, function(c) {
   }));
 
   client.on('close', common.mustCall(function(hadError) {
-    assert.strictEqual(hadError, true, 'Client never errored');
+    assert.strictEqual(hadError, true, `Client error may have occurred. See details below:\n${JSON.stringify(hadError)}`);
```

Comment:
It would be good to just remove the error message instead of adding this individual description.

Reply:
@atvilelas would you be so kind and just remove the message?